app/models/concerns/sdg/relatable.rb

```diff
@@ -26,6 +28,14 @@ def by_sdg_related(sdg_class, code)
 
       joins(sdg_class.table_name.to_sym).merge(sdg_class.where(code: code))
     end
+
+    def revised
+      joins(:sdg_revision)
+    end
+
+    def pending
```

Comment:
This method might be called as `Proposal.pending`, which might be a bit confusing since we might not know what `pending` means, depending on the context. How about `pending_sdg_review`, so it's similar to other scopes like `pending_review` and `pending_flag_review`?

Reply:
I completely agree I did not realize about this!

Reply:
Done!